.github/ISSUE_TEMPLATE/customer-issue.md

```diff
@@ -0,0 +1,37 @@
+---
+name: Customer Issue
+about: Create a report to help us improve
+title: '[CRI]'
+labels: kind/bug, kind/customer-issue
+assignees: ''
+
+---
+
+ **Describe the issue**
+A clear and concise description of what the issue is.
+
+ **To Reproduce**
```

Comment:
why not just

 **Steps to Reproduce**
1.
2.
3.

Reply:
+1